app/views/web_git/commands/status.html.erb

```diff
@@ -25,8 +25,7 @@
           <% end %>
 
           <% if @diff_html.present? %>
-            <iframe srcdoc="<%= @diff_html %>" width="100%" height="400" class="mb-3">
-            </iframe>
+            <%= @diff_html.html_safe %>
```

Comment:
Very interesting that we are no longer using an iframe here! We control the HTML. That's pretty cool.